CleverRaven/Cataclysm-DDA
# Wear Scarves Around Your Head

#### Summary
Content "Scarf Headwraps"

#### Purpose of change
Scarves are long enough to wrap around your head if you're cold, but can't.

#### Describe the solution
This adds a third form to all scarves currently implemented to allow the player to wrap them around their head. The player can activate a scarf to take it from its initial state, to its loose variant, to its headwrap variant, and back to its initial state.

The headwrap variant of scarves provides warmth to the player's head per their warmth in their initial state.

#### Describe alternatives you've considered
The two alternatives to this are to create a crafting recipe the create a headwrap scarf item, or to use EoCs to generate a menu to select the scarf variant. 

The former was deemed too clunky for player experience, while that latter did not scale well for additional scarves added to the game down the line.

#### Testing
1. Start a new game
2. Debug spawn all scarves
3. Activate each scarf, cycling through their initial state, loose state, and headwrap state
4. Results were as expected


#### Additional context
N/A
